workspaces/adventure-pack/goodies/java/src/digits_int/Test.java

```diff
@@ -0,0 +1,72 @@
+package digits_int;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+
+class DigitsIntTest {
+
+  @Test
+  public void withIntegerGreaterThanZero() {
+    try {
+      var digits = AP.digits(4984);
+      assertIterableEquals(
+        IterableIntStream.from(IntStream.of(4, 8, 9, 4)),
+        digits,
+        "Test Failed. Digits generated for positive integer are not correct."
+      );
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail();
+    }
```

Comment:
After we remove the `throws Exception` from the implementation, it should be possible to remove the catches as well.

Reply:
Addressed.